### ISSUE TITLE
@cypress/schematic: ng e2e does not start server

### ISSUE THREAD

**Issue by cexbrayat** [cypress-io/cypress#16514]
### Current behavior

When adding the schematic to a new CLI project, then `ng e2e` does not start the server.

### Desired behavior

`ng e2e` should start the server behind the scene as the `@briebug/cypress-schematic` was (and the built-in Protractor as well).

### Test code to reproduce

```
npx @angular/cli@12.0.0 new cypress-schematic-v12 --defaults
cd cypress-schematic-v12
ng add @cypress/schematic
ng e2e
```

### Versions

Cypress: 7.3.0
`@cypress/schematic`: 1.0.3

**Comment by salemdar** [comment]
I can reproduce this error and it is such a turn off at the moment.

**Comment by cexbrayat** [comment]
@salemdar I forgot to mention it, but as a workaround, you can currently use https://github.com/briebug/cypress-schematic/issues 
The official schematic is just a copy/paste of the other one, so you'll be able to migrate as soon as this issue is resolved.

**Comment by salemdar** [comment]
@cexbrayat that is what I was thinking. Thanks a lot 👍🏻

**Comment by admah** [comment]
@cexbrayat thanks for bringing this up! This was actually an intentional change in our schematic. 

We made it this way so users didn't have to shut Cypress down every time they needed to restart their app server while developing locally since `ng e2e` is meant to be used for the local workflow. If you need to have the server started and tests run, you can always use `ng run {project}:cypress-run` (ie. for CI runs).

Does that work or do you have a reason why we should change the behavior? We're definitely always open to ideas and improvements!

**Comment by cexbrayat** [comment]
@admah Thank you for your answer. 

I think this is a rather surprising change as most schematics work the other way around (default e2e schematic for protractor, the cypress one from Briebug, the one for webdriverio). `ng e2e` usually starts the server and that's what your run on CI (with a `--headless` parameter). Note that `ng e2e` does start a server on a different port if you want to, or if the port is already taken, so it's usually fairly easy to use locally. 

This will probably be a surprise to most Angular developers who migrate from Protractor or from the original cypress schematic. My 2 cents is that I would stick to the default behavior of the ecosystem. 

If you keep it that way, it should probably be clearly stated in the documentation. It was a bit weird for me because the configuration of the `e2e` property in `angular.json` does refer to `devServerTarget` so that was clear in mind that it should start the server. Maybe that's unnecessary then, and would need to be fixed as well.

**Comment by admah** [comment]
Closing this as fixed in `@cypress/schematic`: 1.1.0

**Comment by cexbrayat** [comment]
I gave a try to v1.1.0 and it now works as expected, thanks @admah !

I opened a PR with a tiny possible improvement, see https://github.com/cypress-io/cypress/pull/16565
